CONTRIBUTING.md

```diff
@@ -10,18 +28,23 @@ To find Qbeast issues that make good entry points:
 
 - For issues that require deeper knowledge of one or more technical aspects, look at issues labelled [![help wanted label](https://img.shields.io/github/labels/Qbeast-io/qbeast-spark/help%20wanted)](https://github.com/Qbeast-io/qbeast-spark/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22).
 
-## Developer documentation
-You can find the developer documentation (Scala docs) in the [https://docs.qbeast.io/](https://docs.qbeast.io/).
-
-## Licensing of contributed material
-All contributed code, docs, and other materials are considered licensed under the same terms as the rest of the project. Check [LICENSE](./LICENCE) for more details.
-
 ## Version control branching
 - Always make a new branch for your work, no matter how small
 - Don´t submit unrelated changes to the same branch/pull request
 - Base your new branch off of the appropriate branch on the main repository
 - New features should branch off of the `main` branch
 
+## PRs and Issues
+
+To open and merge PRs, the following is to be respected:
+
+1. **Always open an issue for the PR you're working** with as much detail as possible.
+2. Every PR should have an issue that it is trying to address, ideally one.
+3. The **title of the PR** should follow the schema: Issue <issue-number>: <PR-title>
+4. Ideally, there should be **at least two reviewers per PR**
+5. The author of the PR never gets to merge the PR; **a PR can only be merged by a reviewer**.
+6. Do a **Squash and Merge instead of merge**, again, by a reviewer.
+
```

Comment:
7. Make sure the commit messages are clear and concise and reflect all major changes introduced by the PR.